CI tests should run for nested modules

Currently `cd cmd/genji/shell && go test` fails. This should have been caught by the CI tests.

```
--- FAIL: TestRunTablesCmd (0.00s)
    --- FAIL: TestRunTablesCmd/Table_with_options (0.00s)
        command_test.go:38: 
            	Error Trace:	command_test.go:38
            	Error:      	An error is expected but got nil.
            	Test:       	TestRunTablesCmd/Table_with_options
            	Messages:   	%!(EXTRA bool=true)
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -18,8 +18,8 @@ script:
   - make gen
   - go test -mod vendor -race -cover -coverprofile=coverage.txt -covermode=atomic -timeout=2m ./...
   - go test -mod vendor -race -cover -timeout=2m -tags=tinygo ./...
-  - cd ./cmd/genji && go test -race && cd -
-  - cd ./engine/badgerengine && go test -race && cd -
+  - cd ./cmd/genji && go test -race ./... && cd -
+  - cd ./engine/badgerengine && go test -race ./... && cd -
 
 
 after_success:

